### ISSUE TITLE
Noname Controller does not work (recognized as XBOX 360 by Win10)

### ISSUE THREAD

**Issue by aTr00per** [glutanimate/anjoy-public#6]
Hi,
I'm using a noname controller, which is registered by Win10 as a "Xbox 360 Controller for Windows" and I can use it without fourther installation or configuration. But it does not work with the addon. It gets recognized as "unknown gamepad", but no button work. I tested it in the html5gamepad website without any problems.

Here the debug info as requested:
```
===ANKI===

Anki 2.1.12 (eef86bf3) Python 3.6.7 Qt 5.12.1 PyQt 5.11.3
Platform: Windows 10
Flags: frz=True ao=True sv=1

Add-ons:

ImageResizer
Add Table
Tag Entry Enhancements
Image Occlusion Enhanced Alpha for Anki 21
Search and Replace Tags
Symbols
Export in CSV format
Night Mode
Right Hand Answer Shortcuts (deaktiviert)
Hierarchical Tags Addon 20  21
Toggle Tags via ReviewerEditorBrowser Context Menu
Progress Bar
Card Info During Review (deaktiviert)
Customize Keyboard Shortcuts
Button Colours Good Again
Mini Format Pack
Reset Card Scheduling
Frozen Fields
Multiple Choice for Anki 21
Enhance main window
AnJoy


===ANJOY===

version: 0.1.0-beta.1
[
    {
        "id": "Xbox 360 Controller (XInput STANDARD GAMEPAD)",
        "index": 0,
        "connected": true,
        "timestamp": 1392006.1999999452,
        "mapping": "standard",
        "axes": [
            0.0000152587890625,
            -0.0000152587890625,
            0.0000152587890625,
            -0.0000152587890625
        ],
        "buttons": [
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            },
            {
                "pressed": false,
                "value": 0
            }
        ],
        "vibrationActuator": {
            "type": "dual-rumble",
            "playEffect": "function playEffect() { [native code] }",
            "reset": "function reset() { [native code] }"
        }
    },
    {
        "id": "Unknown Gamepad (Vendor: 06cb Product: 7a13)",
        "index": 1,
        "connected": true,
        "timestamp": 1774.6999999508262,
        "mapping": "",
        "axes": [
            0,
            0
        ],
        "buttons": [
            {
                "pressed": false,
                "value": 0
            }
        ],
        "vibrationActuator": null
    },
    null,
    null,
    4,
    "function item() { [native code] }"
]

Index: 1
```
Hope I could help.

**Comment by glutanimate** [comment]
Hey @aTr00per,

Sorry for taking a while to respond. Across working on PDF Glossary Exporter and a bunch of other 2.1 ports I didn't have a lot of time recently.

I just wanted to thank you for your bug report and let you know that I think I've identified the issue.  The fault lies somewhere between Anki (or rather its underlying web engine), and the add-on – with the web engine recognizing additional gamepads where it shouldn't (the "unknown gamepads"), and the add-on not filtering them out. Other users on Windows seem to be affected by the same issue (especially when using a notebook, as that's where those unknown gamepads seem to appear the most).

I've created a new bug report in #7 where I will track the progress of the fix moving forward. I'm planning on including the fix in the upcoming release of the add-on which should be in a couple of weeks or so.

Thanks again for your support, and for taking the time to help make AnJoy better!
